proto/tribune_tts.proto

```diff
@@ -48,13 +60,70 @@ message SynthesizeRequest
 // Provides information to the synthesizer that specifies how to process the request.
 message SynthesizeConfig
 {
-    // Desired sampling frequency in hertz of synthesized audio.
+    // Desired sampling frequency in hertz of synthesized audio. The value 0 means use the default Synthesizer sampling rate.
```

Comment:
This field could be moved to `AudioConfig` message

Reply:
Moved.